common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.py

```diff
@@ -78,6 +79,16 @@ def _generate_artifacts(model: def_types.Model, save_dir: pathlib.Path,
 
     print(f"Completed generating artifacts {model.name}\n")
 
+    if auto_upload:
+      subprocess.run(
```

Comment:
Maybe extract this into a common place as it is reused 3 times?

Reply:
Moved to utils.py.